.github/workflows/backend.yml

```diff
@@ -213,19 +213,27 @@ jobs:
         with:
           app_id: ${{ vars.SENTRY_INTERNAL_APP_ID }}
           private_key: ${{ secrets.SENTRY_INTERNAL_APP_PRIVATE_KEY }}
+
       - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
+      - uses: astral-sh/setup-uv@884ad927a57e558e7a70b92f2bccf9198a4be546 # v6
+        with:
+          version: '0.8.2'
+
       - uses: getsentry/action-setup-venv@3a832a9604b3e1a4202ae559248f26867b467cc7 # v2.1.1
         with:
           python-version: 3.13.1
           cache-dependency-path: requirements-dev-frozen.txt
-          install-cmd: pip install -q --constraint requirements-dev-frozen.txt pip-tools
+          install-cmd: uv pip install -q --constraint requirements-dev-frozen.txt pip-tools
+
       - name: check requirements
         run: |
-          python -S -m tools.freeze_requirements
+          make freeze-requirements
```

Comment:
this previously intentionally skilled `make` because it went through all the shell startup which installed a bunch of stuff not wanted in CI